R/pipgd_lorenz.R

```diff
@@ -141,33 +152,30 @@ pipgd_validate_lorenz <-
 #' @param params list of parameters from `pipgd_validate_lorenz()`
 #' @param mean numeric scalar of distribution mean. Default is 1
 #'
-#' @return list of values with best lorenz fit for distributional Stats
+#' @return Returns a list of values with the best lorenz fit given the
+#' distributional Stats.
+#'
 #' @export
 #'
 #' @examples
-#' # Using Lorenz parameters from get_gd_lorenz_params
-#' withr::local_options(pipster.return_complete  = TRUE)
-#' params <- pipgd_validate_lorenz(
-#'   welfare = pip_gd$L,
-#'   weight = pip_gd$P)
-#'
-#' params <- pipgd_validate_lorenz(
-#'   params = params,
-#'   complete = TRUE)
-#' pipgd_select_lorenz(params = params)
+#' # Example 1: Directly using welfare and weight vectors
+#' pipgd_select_lorenz(welfare = pip_gd$L,
+#'                     weight = pip_gd$P)
 #'
-#' # Using Lorenz parameters from pipgd_validate_lorenz
-#' params <- pipgd_validate_lorenz(
-#'   welfare = pip_gd$L,
-#'   weight = pip_gd$P,
-#'   complete = TRUE)
-#' pipgd_select_lorenz(params = params)
+#' # Example 2: Specifying mean and poverty line
+#' custom_mean <- mean(pip_gd$L)
```

Comment:
`L` is cummulative welfare share, so this `mean` does not apply. It should a proper mean either in local currency units or in PPP values.